prow/cmd/deck/BUILD

```diff
@@ -1,5 +1,29 @@
 package(default_visibility = ["//visibility:public"])
 
+load("@io_bazel_rules_docker//container:container.bzl", "container_image")
+
+container_image(
+    name = "asset-base",
+    base = "@alpine-base//image",
+    # With paths relative to the current directory.
+    data_path = ".",
+    # Add files under into the root directory.
+    directory = "/",
+    # Everything under static/
+    files = glob(["static/**/*"]),
+)
+
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+go_image(
+    name = "image",
+    base = ":asset-base",
+    embed = [":go_default_library"],
+    importpath = "k8s.io/test-infra/prow/cmd/deck",
```

Comment:
I kinda wish there were a way `go_image` could import the existing `go_binary` rule here, rather than duplicating all of the attributes.

Reply:
Ideally we could just treat `go_image` as the `go_binary`, but because of https://github.com/bazelbuild/rules_go/issues/748 Gazelle always adds the `go_binary` target back.

We could probably create a form that takes a pre-existing binary target, but don't have one today.

Reply:
FWIW, you can always use `:foo.binary` for the binary target (if needed).